internal/cmd/platform/delete_platform.go

```diff
@@ -61,14 +58,16 @@ func (dpc *DeletePlatformCmd) Validate(args []string) error {
 // Run runs the command's logic
 func (dpc *DeletePlatformCmd) Run() error {
 	fieldQuery := util.GetResourceByNamesQuery(dpc.names)
-	err := dpc.Client.DeletePlatformsByFieldQuery(fieldQuery)
-	if respErr, ok := err.(errors.ResponseError); ok && respErr.StatusCode == http.StatusNotFound {
-		output.PrintMessage(dpc.Output, "Platform(s) not found.\n")
-		return nil
-	} else if err != nil {
+	if err := dpc.Client.DeletePlatformsByFieldQuery(fieldQuery); err != nil {
+		if strings.Contains(err.Error(), "StatusCode: 404") {
```

Comment:
It's not the ideal check to look for a specific string, but I can't think of another way when we don't have `errors.ResponseError` anymore